bundles/org.openhab.binding.worxlandroid/README.md

```diff
@@ -0,0 +1,649 @@
+<img src="images/landroid.svg">
+
+# WorxLandroid Binding
+
+This is the binding for Worx Landroid robotic lawn mowers. It connects openHAB with your WorxLandroid mower using the API and MQTT. This binding allows you to integrate, view and control supported Worx lawn mowers in the openHAB environment.
+
+## Supported Things
+
+Currently following Things are supported:
+
+- **Bridge Worx Landroid API** Thing representing the handler for Worx API
+- One or many Things for supported **Landroid Mower**'s
+
+## Discovery
+
+A Bridge is required to connect to the Worx API. Here you can provide your credentials for your WorxLandroid account. Once the Bridge has been added Worx Landroid mowers will be discovered automatically.
+
+## Binding Configuration
+
+Following options can be set for the **Bridge Worx Landroid API**:
```

Comment:
The following options...

Can you also add the thing id `bridge` at the end?

Reply:
Done